src/app/content/components/NudgeStudyTools/utils.ts

```diff
@@ -42,7 +44,11 @@ export const getPositions = (target: HTMLElement, isMobile: boolean, windowWidth
   const spotlightHeight = height + (padding * 2);
   const spotlightWidth = width + (padding * 2);
   const arrowTopOffset = spotlightTopOffset + spotlightHeight + remsToPx(arrowTopMargin);
-  const arrowLeft = spotlightLeftOffset + remsToPx(arrowLeftMargin);
+  // right edge of arrow image should be on the middle of the spotlight (adjusted for a margin)
+  const centerPoint = spotlightLeftOffset + (spotlightWidth / 2);
+  const arrowWidth = remsToPx(isMobile ? arrowMobileWidth : arrowDesktopWidth);
+  const arrowLeft = centerPoint - arrowWidth + remsToPx(arrowLeftMargin);
```

Comment:
I imagine that if you had a bounding box of the spotlight, then you could probably position all of the other elements with plain css, but I'm not sure if it's worth changing.  Up to you which approach you think is better.